server/src/api/model.js

```diff
@@ -78,10 +60,11 @@ function setLocation(id, time, longitude, latitude, callback) {
  */
 function getPrice(longitude, latitude, callback) {
     log.trace(module, getPrice);
-    // Longitudes and latitudes in the DB are precise to 13 decimal places, which is 6.5 nm. To fix
-    // this, we allow a tolerance of 10^-4 degrees, which is about 11 m. We then log how many
-    // results this returns (for debugging) and return the first (and hopefully only) result.
-    const R10M   = 1e-15;
+    // Longitudes and latitudes in the DB are precise to 13 decimal places. 10^-12 of a degree is
+    // about 6.5 nm, which means the server won't return anything unless the user stands in a very
+    // specific spot. To fix this, we allow a tolerance of 10^-6 degrees, which is about 11 m. We
+    // then log how many results this returns (for debugging) and return the first (and hopefully only) result.
+    const R10M   = 1e-6;
```

Comment:
Is this a requirement for any requests? What happens if we submit outside this range?